### PR TITLE
Some docs and annotation fix 

### PR DESCRIPTION
**Pull request by zz-bmj**
Some docs fix

### CODE REVIEW

**Comment by zz-bmj** [comment]
the docs need a big fix, we will make it after this stage of development

File: api/rest/version.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func listVersions(request *restful.Request, response *restful.Response) {
 }
 
 //
-// POST: /api/v0.1/:uid/versions/:versionID/cancelbuild
+// POST: /api/v0.1/:uid/versions/:versionID/cancelbuildcancelbuild
```
Comment (supereagle) [inline api/rest/version.go]: This has no error, no need to change. Right?

File: docs/developer-guide.md [hunk 2]
```diff
@@ -38,14 +38,21 @@ And Our travis CI jobs would also run these, so you could also check out the bui
...
-In the future we would run these tests by cyclone.
+then, we would run these tests by cyclone.
 
+If you deploy with k8s, you could run e2e test by typing:
+
+```shell
+./scripts/e2e-k8s.sh
+```
+
+then, we would run these tests by cyclone.
```
Comment (supereagle) [inline docs/developer-guide.md]: A newline is needed here.

File: scripts/tools/api.py [hunk 3]
```diff
@@ -69,7 +69,7 @@ def create_service(user_id, service_name):
         "username": "caicloud",
         "repository": {
             #"url": "/home/superxi/gopath/src/github.com/caicloud/console-web",
-            "url": "https://github.com/zoumo/go_test.git",
+            "url": "https://github.com/zz-bmj/go-demo",
```
Comment (supereagle) [inline scripts/tools/api.py]: The new repo is just for your test? Whether the old repo still can work? If it still can work, no need to change it.

File: docs/developer-guide.md [hunk 4]
```diff
@@ -38,14 +38,21 @@ And Our travis CI jobs would also run these, so you could also check out the bui
 
 Now cyclone's e2e test would start a local circle server and register local docker host as a worker to circle server. Then it would start a standalone process to run the e2e test cases by sending RESTful requests.
 
-You could run e2e test by typing:
+If you deploy with docker, you could run e2e test by typing:
```
Comment (supereagle) [inline docs/developer-guide.md]: Does the e2e test already support to be run by cyclone? If so, maybe it is better to change the description like this:
The e2e test can be run by cyclone itself. If you deploy with docker, you could run it by typing:
```shell
./scripts/e2e-docker.sh
```
If you deploy with k8s, you could run it by typing:
```shell
./scripts/e2e-k8s.sh
```

File: Jenkinsfile [hunk 5]
```diff
@@ -111,7 +110,7 @@ podTemplate(
 
                         echo "buiding worker"
                         go build -i -v -o cyclone-worker github.com/caicloud/cyclone/cmd/worker 
-                        docker build -t ${WORKER_IMAGE} -f Dockerfile.worker .
+                        docker build -t cargo.caicloudprivatetest.com/caicloud/cyclone-worker:e2e -f Dockerfile.worker .
```
[the remark this answers is not in the document]
Comment (zz-bmj) [reply Jenkinsfile]: the env can't get the tag `:e2e`

File: api/rest/version.go (hunk 1 above)
Comment (zz-bmj) [reply api/rest/version.go]: the word `cancelbuild `  repeat twice, it's a small mistake?
image

File: docs/developer-guide.md (hunk 4 above)
Comment (zz-bmj) [reply docs/developer-guide.md]: OK, your description is much better than mine, I will fix it with your suggestion.

File: scripts/tools/api.py (hunk 3 above)
Comment (zz-bmj) [reply scripts/tools/api.py]: the old repo name has been changed, so I fork it from "zoumo" and then change it with my repo.

File: docs/developer-guide.md (hunk 2 above)
Comment (zz-bmj) [reply docs/developer-guide.md]: done~

**Comment by zz-bmj** [comment]
@supereagle It's would be nice of you if you make a review~

**Comment by supereagle** [comment]
LGTM

**Pull request merged**